### DISCUSSION TITLE
Need some help :)

### DISCUSSION THREAD

**Discussion by lassejlv** [cdnjs/cdnjs#14090] [General] [answered]
hey, i need help to how i can add a libary to cdnjs 😄

**Comment by MattIPv4** [comment, marked as answer]
Hey!

Sorry for the delayed response. To add a new library you'll need to open a PR in https://github.com/cdnjs/packages to add the JSON configuration file for your library.

I recommend taking a look at some recently added libraries there through the latest commits to get a sense of what needs to be in the config file. The [CONTRIBUTING.md](https://github.com/cdnjs/packages/blob/master/CONTRIBUTING.md) file also goes over specific bits of the file, such as the required auto-update configuration, as well as some baseline requirements in terms of the popularity of the package.

Hope that helps.
